package.json

```diff
@@ -70,6 +71,14 @@
     "preios:xcode10": "cd node_modules/react-native && ./scripts/ios-install-third-party.sh && cd third-party/glog-0.3.5 && [ -f libglog.pc ] || ../../scripts/ios-configure-glog.sh",
     "ios": "react-native run-ios",
     "test": "cross-env NODE_ENV=test node node_modules/jest/bin/jest.js --verbose --config jest.config.js",
+    "device-tests":"cross-env NODE_ENV=test node node_modules/jest/bin/jest.js --detectOpenHandles --verbose --config jest_ui.config.js",
```

Comment:
We can use `jest` here instead of `node node_modules/jest/bin/jest.js`, yarn can look into `node_modules/.bin`

Reply:
👍🏽